<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeoFEA</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>851</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1229</width>
      <height>849</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="textbox_FilenameLDR">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>715</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
	
    <widget class="QPushButton" name="button_SelectLDR">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select LDR</string>
     </property>
    </widget>
    
	<widget class="QPushButton" name="button_LoadLDR">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Analysis</string>
     </property>
    </widget>
	
	
    <widget class="QComboBox" name="combo_Analysis">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
	
	
    <widget class="QPushButton" name="button_startFEA">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>560</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>FEA</string>
     </property>
    </widget>
    
	 <widget class="QLabel" name="label_work_dir">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>580</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Work directory</string>
     </property>
    </widget>
	
	 <widget class="QLineEdit" name="textbox_workdir">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>580</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>My work dir</string>
     </property>
    </widget>

    <widget class="QPushButton" name="button_SelectWorkdir">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>580</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Directory</string>
     </property>
    </widget>

    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>670</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Postprocess</string>
     </property>
    </widget>
	
	<widget class="QPushButton" name="button_viewPNG">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>710</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View PNG</string>
     </property>
    </widget>

    <widget class="QPushButton" name="button_openParaview">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>750</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Paraview</string>
     </property>
    </widget>

    <widget class="QPushButton" name="button_checkLimitValues">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>790</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Limits</string>
     </property>
    </widget>


    <widget class="QPushButton" name="button_openLeoCAD">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>131</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Open LeoCAD</string>
     </property>
    </widget>
    
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Jobname</string>
     </property>
    </widget>
    
    <widget class="QLineEdit" name="textbox_jobname">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>861</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load scale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textbox_gravityScale">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetLDR">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>711</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>503</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Mass = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textbox_mass">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>500</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>503</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of segments = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textbox_numberBricks">
     <property name="geometry">
      <rect>
       <x>715</x>
       <y>500</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>503</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Costs = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textbox_costs">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>500</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_force">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate forces (Fx,Fy,Fz)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_contact">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate Contact</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CodeAster">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Code_Aster</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>


    <widget class="QTextEdit" name="textEdit_resultsMinForce">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>670</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
	
	
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>670</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. tensile Force</string>
     </property>
    </widget>
	
	
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. Force</string>
     </property>
    </widget>
	
    <widget class="QLineEdit" name="textbox_maxForce">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textbox_maxDisplacement">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. Displacement</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>790</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. Displacement</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>750</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Min. Displacement</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>790</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMinDisplacement">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>750</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxForce">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>710</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>710</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. compressive Force</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxForce_p">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>710</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMinForce_p">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>670</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement_p">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>790</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMinDisplacement_p">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>750</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement_m">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>790</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMinDisplacement_m">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>750</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMaxForce_m">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>710</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_resultsMinForce_m">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>670</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>640</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Point</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
